e2e/phone-field.e2e-spec.ts

```diff
@@ -0,0 +1,90 @@
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('xs');
+    expect('#screenshot-phone-field').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-xs'
+    });
+  });
+
+  it('should match previous phone field screenshot when touched', (done) => {
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+    element(by.css('#screenshot-phone-field .sky-form-control')).click();
+    expect('#screenshot-phone-field').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-touched'
+    });
+  });
+
+  it('should match previous phone field screenshot when invalid', (done) => {
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+    let inputElement = element(by.css('#screenshot-phone-field .sky-form-control'));
+    inputElement.click();
+    inputElement.sendKeys('1234');
+    element(by.css('body')).click();
+    expect('#screenshot-phone-field').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-invalid'
+    });
+  });
+
+  it('should match previous phone field screenshot when dropdown is open', (done) => {
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+    let dropdownElement = element(by.css('#screenshot-phone-field .sky-dropdown-button'));
+    dropdownElement.click();
+    SkyHostBrowser.moveCursorOffScreen();
+    expect('.sky-popover-container.sky-popover-placement-fullscreen').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-dropdown'
+    });
+  });
+
+  it('should match previous phone field screenshot', (done) => {
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+    expect('#screenshot-phone-field-default').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-default'
+    });
+  });
+
+  it('should match previous phone field screenshot (screen: xs)', (done) => {
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('xs');
+    expect('#screenshot-phone-field-default').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-default-xs'
+    });
+  });
+
+  it('should match previous phone field screenshot when dropdown is open with default', (done) => {
+    SkyHostBrowser.get('visual/phone-field');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+    let dropdownElement = element(by.css('#screenshot-phone-field-default .sky-dropdown-button'));
+    dropdownElement.click();
+    let dropdownItemElement = element(by.css('#screenshot-phone-field-default .iti-flag.al'));
+    dropdownItemElement.click();
+    dropdownElement.click();
+    SkyHostBrowser.moveCursorOffScreen();
+    expect('.sky-popover-container.sky-popover-placement-fullscreen').toMatchBaselineScreenshot(done, {
+      screenshotName: 'phone-field-default-dropdown'
+    });
+  });
+});
```

Comment:
I noticed there's several missing tests for small screens. Is that intentional?

Reply:
Ya. I only added tests for states where the size would matter. So I didn't do small screen shots for open dropdowns as that would be covered in the dropdown visual tests. I also didn't add tests for the invalid and valid state stylings as those won't change on small screens and should be covered by the theme tests that cover form control stylings.

Reply:
sounds good.